packages/tools/src/stateManagement/segmentation/polySeg/Labelmap/convertContourToLabelmap.ts

```diff
@@ -0,0 +1,275 @@
+import {
+  StackViewport,
+  Types,
+  VolumeViewport,
+  cache,
+  utilities,
+  getWebWorkerManager,
+  volumeLoader,
+  imageLoader,
+  metaData,
+  Enums,
+  triggerEvent,
+  eventTarget,
+} from '@cornerstonejs/core';
+import {
+  Annotation,
+  ContourSegmentationData,
+  PolySegConversionOptions,
+} from '../../../../types';
+import { getAnnotation } from '../../..';
+import { vec3 } from 'gl-matrix';
+import { Events } from '../../../../enums';
+
+const workerManager = getWebWorkerManager();
+
+export async function convertContourToVolumeLabelmap(
+  contourRepresentationData: ContourSegmentationData,
+  options: PolySegConversionOptions = {}
+) {
+  const { viewport } = options;
+
+  let imageIds;
+  if (viewport instanceof VolumeViewport) {
```

Comment:
Could you move this into a utility function to abstract the idea of getting a set of imageIds?